python/mxnet/libinfo.py

```diff
@@ -29,6 +29,9 @@ def find_lib_path():
         dll_path.extend([p.strip() for p in os.environ['LD_LIBRARY_PATH'].split(":")])
     if os.name == 'nt':
         dll_path = [os.path.join(p, 'libmxnet.dll') for p in dll_path]
+    elif platform.system() == 'Darwin':
+        dll_path = [os.path.join(p, 'libmxnet.dylib') for p in dll_path]+ \
```

Comment:
I think the produced suffix of shared library depends on the tool-chain you use. In my case, I use XCode with clang compiler configured with CMake, it constantly produce the shared library with .dylib suffix.